buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -0,0 +1,158 @@
+@file:Suppress("unused", "SpellCheckingInspection")
+
+typealias DependencyItem = CharSequence
+
+object Versions {
+    const val Kotlin = "1.3.40"
+    const val Reaktor = "1.0.17"
+    const val AndroidAppUtil = "14"
+    const val AndroidX = "1.1.0-beta01"
+    const val Navigation = "2.0.0"
+    const val Koin = "2.0.1"
+    const val RxBinding = "3.0.0-alpha2"
+    const val Lifecycle = "2.0.0"
+    const val Retrofit = "2.6.0"
+    const val OkHttp = "4.0.0"
+    const val Leakcanary = "1.6.3"
+    const val Glide = "4.9.0"
+    const val Mockito = "2.28.2"
+    const val Powermock = "2.0.2"
+    const val Espresso = "3.1.0"
+    const val AndroidXTest = "1.1.0"
+}
+
+object BuildScriptDependencies {
+    const val AndroidBuildTools = "com.android.tools.build:gradle:3.4.1"
+    const val KotlinPlugin = "org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.Kotlin}"
+    const val NavigationSafeArgs = "androidx.navigation:navigation-safe-args-gradle-plugin:${Versions.Navigation}"
+
+    const val DependencyVersions = "com.github.ben-manes:gradle-versions-plugin:0.21.0"
+    const val EasyLauncher = "com.akaita.android:easylauncher:1.3.1"
+    const val GraphViz = "guru.nidi:graphviz-java:0.5.3"
+    const val DependencyGraph = "com.vanniktech:gradle-dependency-graph-generator-plugin:0.5.0"
+}
+
+object ModuleDependencies {
+    const val KotlinStd = "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${Versions.Kotlin}"
+
+    const val Reaktor = "at.florianschuster.reaktor:reaktor-android-koin:${Versions.Reaktor}"
+
+    const val Gson = "com.google.code.gson:gson:2.8.5"
+    const val Timber = "com.jakewharton.timber:timber:4.7.1"
+    const val Threeten = "com.jakewharton.threetenabp:threetenabp:1.2.1"
+
+    const val LeakCanaryOp = "com.squareup.leakcanary:leakcanary-android:${Versions.Leakcanary}"
+    const val LeakCanaryNoOp = "com.squareup.leakcanary:leakcanary-android-no-op:${Versions.Leakcanary}"
+}
+
+enum class AndroidAppUtil(val dep: String): DependencyItem by dep {
```

Comment:
I would just like to bring up the discussion whether we want to keep the val name short as `dep`, or if we could spare a `dependency` here.. :thinking:

(I'm not sure which one I would consider better, but as `dep` is (at least for me) not that familiar as an abbreviation to `dependency`, I'm slightly tending to `dependency`...)

```kotlin
enum class AndroidAppUtil(val dependency: String): DependencyItem by dependency {
    UI("com.tailoredapps.androidutil:util-ui:${Versions.AndroidAppUtil}");
    
    override fun toString() = dependency
}
```
vs.
```kotlin
enum class AndroidAppUtil(val dep: String): DependencyItem by dep {
    UI("com.tailoredapps.androidutil:util-ui:${Versions.AndroidAppUtil}");
    
    override fun toString() = dep
}
```

Reply:
ok